.github/CODEOWNERS

```diff
@@ -0,0 +1,192 @@
+# This file lists the owners of code in different areas of the lnd codebase
```

Comment:
added

Reply:
So I don't think it needs to be 100% exhaustive. Otherwise, those ppl will be added each time a contributor makes a typo fix in a proto. Certain packages like `build`, have no true owner, and instead we can rely on the load balancing option for code review. 

If we want to chop up the sub-server packages/files, then I think that could make sense.

Reply:
So far, I haven't included docs, input, labels, lnmock, make, scripts, subscribe and tools.
If any of these should be included, or any that you think is best excluded so that reviews are assigned via load balancing option for those, do let me know.